### PR TITLE
Adds MANIFEST to specify what files to distribute with package

### PR DESCRIPTION
**Pull request by dodijk**
#60 was not the solution for our PyPI issues. The files are actually not distributed with the textpipe package. Adding a `MANIFEST` file to specify this. See: https://docs.python.org/3/distutils/sourcedist.html#the-manifest-in-template

### CODE REVIEW

File: MANIFEST [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CONTRIBUTORS.md
+LICENSE
+README.md
+requirements.txt
+VERSION
```
Comment (lmdehaas) [inline MANIFEST]: Missing newline

**Comment by dodijk** [comment]
I tried `python setup.py install`. Then `textpipe-0.3.2-py3.7/EGG-INFO/SOURCES.txt` includes the files with the `MANIFEST.in` and not without.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.3.1
+0.3.2
```
Comment (lmdehaas) [inline VERSION]: No newline

Comment (dodijk) [reply VERSION]: That is on purpose. I got the message:
```
/usr/local/lib/python3.7/site-packages/setuptools/dist.py:398: UserWarning: Normalizing '0.3.2
' to '0.3.2'
  normalized_version,
```

Comment (lmdehaas) [reply VERSION]: 👍

**Comment by dodijk** [comment]
```
$ pip3 install -v --index-url https://test.pypi.org/simple/ textpipe
...
Successfully installed textpipe-0.3.2
```

**Pull request merged**